guides/advanced_guides/settings.md

```diff
@@ -0,0 +1,251 @@
+
+#### 5. Words Position
+
+Results are sorted by **the position of the query words in the attributes**: find documents that contain query terms earlier in their attributes first.
+
+#### 6. Exactness
+
+Results are sorted by **the similarity of the matched words with the query words**: find documents that contain exactly the same terms as the ones queried first.
+
+[Learn more about ranking rules](/guides/main_concepts/relevancy.md)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "rankingRules": [
+          "typo",
+          "words",
+          "proximity",
+          "attribute",
+          "wordsPosition",
+          "exactness",
+          "asc(release_date)",
+          "desc(rank)"
+      ]
+  }'
+```
+
+## Distinct attribute
+
+The **value of a field** whose attribute is set as a distinct attribute will always be **unique** in the returned documents.
+
+`distinctAttribute=<String>`
+
+- `<String>` (String, defaults to `null`)
+
+  The field name.
+
+[Learn more about the distinct attribute](/guides/advanced_guides/distinct.md)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "distinctAttribute": "movie_id"
+  }'
+```
+
+## Searchable attributes
+
+Fields in which to **search for matching query words**.
+
+`searchableAttributes=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, defaults to all attributes found in the documents)
+
+  An array of strings that contains searchable attributes sorted by order of importance (arranged from the most important attribute to the least important attribute).
+
+[Learn more about searchable attributes](/guides/advanced_guides/field_properties.md#searchable-fields)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "searchableAttributes": [
+          "uid",
+          "movie_id",
+          "title",
+          "description",
+          "poster",
+          "release_date",
+          "rank"
+      ]
+  }'
+```
+
+## Displayed attributes
+
+Displayed attributes are the **fields contained in each matching document**.
+
+Documents returned upon search contain only displayed fields.
+
+`displayedAttributes=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, defaults to all attributes found in the documents)
+
+  An array of strings that contains attributes of an index to display.
+
+[Learn more about displayed attributes](/guides/advanced_guides/field_properties.md#displayed-attributes)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "displayedAttributes": [
+          "title",
+          "description",
+          "poster",
+          "release_date",
+          "rank"
+      ]
+  }'
+```
+
```

Comment:
Could we add an explaination to this example? For example by removing a field and explaining that this field will not be in the returned documents